core/src/test/java/com/radixdlt/benchmark/TxnCommitAndReadBenchmarkTest.java

```diff
@@ -190,9 +193,13 @@ private void readAllTxnsThroughLedgerSync(DeterministicTest test) {
         totalTxnReaderCalls);
   }
 
-  private List<RawLedgerTransaction> createUniqueTransactions(int numTransactions) {
+  private List<RawLedgerTransaction> createUniqueTransactions(
+      int numTransactions, int notaryPrivKeySeed) throws PrivateKeyException, PublicKeyException {
     final List<RawLedgerTransaction> res = new ArrayList<>();
-    final var notary = ECKeyPair.generateNew();
+    final byte[] prvBytes = new byte[ECKeyPair.BYTES];
+    final byte[] seedBytes = Longs.toByteArray(notaryPrivKeySeed + 1);
+    System.arraycopy(seedBytes, 0, prvBytes, prvBytes.length - seedBytes.length, seedBytes.length);
+    final var notary = ECKeyPair.fromPrivateKey(prvBytes);
```

Comment:
Out of interest, what was the bug with the previous version? Was `generateNew()` not sufficient random to guarantee unique keys (and so unique transactions?)

Reply:
not really a bug, but it should by slightly faster and fair when comparing results